docs/digital-object-full-view.md

```diff
@@ -0,0 +1,32 @@
+# View Modes: Digital Object Full View
+
+`View modes` tell Archipelago Digital Objects (ADOs) how to be displayed to the world. This documentation will give a brief overview of the View mode `Digital Object Full View` and how it can be used when creating or editing ADOs.
+
+## Digital Object Full View
+The `Digital Object Full View` is the default `View mode` used for the Media type `Photograph` and any other Media type that is not already configured to a View mode*. `Digital Object Full View` has the ability to zoom in, zoom out, full screen, rotate and display multiple images associated with an ADO.
+
+<details><summary>* Configuring ADO Types to View modes </summary>
+<div>
+
+The **ADO Type to View mode Mapping** page can be accessed at */admin/config/archipelago/viewmode_mapping*.
+This page will allow users to configure which `View mode` is used by default via the JSON key `type`.
+
+![ADO to View Mode Mapping](../imgs/digital-object-full-view/00_ado-type-to-view-mode-mapping.jpg)
+
+</div>
+</details></p>
+
+![Image Viewer](../imgs/digital-object-full-view/01_imageviewer.jpg)
+
+## Manage display
+In the Webform there is a section titled `DISPLAY SETTINGS` where users will be able to edit which `View mode` is being used for the object. To edit the `View mode`, go to the `Manage display` tab at the top of the Webform.
```

Comment:
Nope. its not in the Webform.. its in the Ingest form (the webform being only the metadata part). Its part of the Form Mode where the webform is the widget for the SBF and the Display Setting another widget...